spec/FrameSpec.js

```diff
@@ -0,0 +1,158 @@
+describe("Frame", function() {
+  var frame;
+
+  beforeEach(function() {
+    frame = new Frame();
+  });
+
+  describe('playing', function(){
```

Comment:
Descriptions inside the `describe` and `it` blocks could be a bit clearer/ more meaningfull